### ISSUE TITLE
"declare const Tts = new ReactNativeTts()" gives TypeScript error "A 'const' initializer in an ambient context must be a string or numeric literal or literal enum reference."

### ISSUE THREAD

**Issue by ngocdaothanh** [ak1394/react-native-tts#172]
Line 80 of https://github.com/ak1394/react-native-tts/blob/master/index.d.ts

```
declare const Tts = new ReactNativeTts();
```

gives error:

```
A 'const' initializer in an ambient context must be a string or numeric literal or literal enum reference.
```

TypeScript: 4.2.2

**Comment by ngocdaothanh** [comment]
I think the fix is changing the code to:

```ts
declare const Tts: ReactNativeTts;
```

**Comment by ak1394** [comment]
Thanks! It should be fixed in 4.0.0 release.
